tests/contrib/test_multistore_file.py

```diff
@@ -48,6 +48,24 @@ def filename(self):
         return self.filename_str
 
 
+class Test__dict_to_tuple_key(unittest.TestCase):
```

Comment:
This test module [still uses][1] `unittest` (i.e. has no need for backports). Uses of `unittest2` have been on an as-needed basis. You think it's worth switching over wholesale?

[1]: https://github.com/google/oauth2client/blob/1f1821653d63c68e657965da4525f04a846c18d0/tests/contrib/test_multistore_file.py#L22